### PR TITLE
Migrate from 1INCH to SUSHI

### PR DESCRIPTION
**Pull request by drondin**
Strats were migrated to Sushi HODL

### CODE REVIEW

File: src/main/java/pro/belbix/ethparser/web3/contracts/HarvestVaultAddresses.java [hunk 1]
```diff
@@ -57,10 +57,10 @@ private HarvestVaultAddresses() {
       new Contract(11608456, "SUSHI_MIC_USDT", "0x6F14165c6D529eA3Bfe1814d0998449e9c8D157D"),
       new Contract(11608466, "SUSHI_MIS_USDT", "0x145f39B3c6e6a885AA6A8fadE4ca69d64bab69c8"),
       new Contract(11639716, "CRV_OBTC", "0x966A70A4d3719A6De6a94236532A0167d5246c72"),
-      new Contract(11647784, "ONEINCH_ETH_DAI", "0x8e53031462e930827a8d482e7d80603b1f86e32d"),
-      new Contract(11647839, "ONEINCH_ETH_USDC", "0xd162395c21357b126c5afed6921bc8b13e48d690"),
-      new Contract(11647866, "ONEINCH_ETH_USDT", "0x4bf633a09bd593f6fb047db3b4c25ef5b9c5b99e"),
-      new Contract(11647897, "ONEINCH_ETH_WBTC", "0x859222dd0b249d0ea960f5102dab79b294d6874a"),
+      new Contract(11647784, "ONEINCH_ETH_DAI", "0x29EC64560ab14d3166222Bf07c3F29c4916E0027"),
```
Comment (belbix) [inline src/main/java/pro/belbix/ethparser/web3/contracts/HarvestVaultAddresses.java:60]: we will ruin history parsing if change old vaults
unfortunately, need to create new
use _V1 postfix

Comment (drondin) [reply src/main/java/pro/belbix/ethparser/web3/contracts/HarvestVaultAddresses.java:60]: sure, do we want to change the name to SUSHI or keep ONEINCH?

Comment (drondin) [reply src/main/java/pro/belbix/ethparser/web3/contracts/HarvestVaultAddresses.java]: I renamed to SUSHI for redability

Comment (drondin) [reply src/main/java/pro/belbix/ethparser/web3/contracts/HarvestVaultAddresses.java]: I just noticed this strategies were already present, closing PR

```
      new Contract(11999019, "ST_ETH_DAI_HODL", "0xF5833723b150929D1Fddf785ED9D92eEe722387d"),
      new Contract(11999015, "ST_ETH_USDC_HODL", "0x378C314028071C92efE15d6990B6cf93594fCB9D"),
      new Contract(11998997, "ST_ETH_USDT_HODL", "0x0c67FBa277A3FE1B0a792ef5bc798cBbDA15a7f5"),
      new Contract(11999022, "ST_ETH_WBTC_HODL", "0x08aA65118996eaa61372B65978Cfa684F2C749b2"),
```

**Comment by drondin** [comment]
I just noticed this strategies were already present

```
      new Contract(11999019, "ST_ETH_DAI_HODL", "0xF5833723b150929D1Fddf785ED9D92eEe722387d"),
      new Contract(11999015, "ST_ETH_USDC_HODL", "0x378C314028071C92efE15d6990B6cf93594fCB9D"),
      new Contract(11998997, "ST_ETH_USDT_HODL", "0x0c67FBa277A3FE1B0a792ef5bc798cBbDA15a7f5"),
      new Contract(11999022, "ST_ETH_WBTC_HODL", "0x08aA65118996eaa61372B65978Cfa684F2C749b2"),

```

**Pull request closed without merging**